com.tomjwatson.Emote.yml

```diff
@@ -0,0 +1,68 @@
+app-id: com.tomjwatson.Emote
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: emote
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=xdg-config/autostart:create
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - "*.la"
+  - "*.a"
+modules:
+  - flatpak/python3-requirements.json
+  - name: keybinder-3.0
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+  - name: xdotool
+    no-autogen: true
+    make-args:
+      - WITHOUT_RPATH_FIX=1
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+      - INSTALLMAN=${FLATPAK_DEST}/share/man
+    sources:
+      - type: archive
+        url: https://github.com/jordansissel/xdotool/archive/v3.20211022.1/xdotool-3.20211022.1.tar.gz
+        sha256: 82b15a944a5e82fee15e0f6116bd9f642bc3d0bb6989fc0ca5ad9dfe35de0847
+        x-checker-data:
+          type: anitya
+          project-id: 8648
+          stable-only: true
+          url-template: https://github.com/jordansissel/xdotool/archive/v$version/xdotool-$version.tar.gz
+  - name: wl-clipboard
+    buildsystem: meson
+    config-opts:
+      - -Dzshcompletiondir=no
+      - -Dfishcompletiondir=no
+    sources:
+      - type: archive
+        url: https://github.com/bugaevc/wl-clipboard/archive/v2.1.0.tar.gz
+        sha256: 72dab9a7d3835c76d6ff2089f15ffec9e064a321e5f3cbbe961a8fa81aff5573
+        x-checker-data:
+          type: anitya
+          project-id: 49082
+          url-template: https://github.com/bugaevc/wl-clipboard/archive/v$version.tar.gz
+    cleanup:
+      - /share/man
+      - /share/bash-completion
+  - name: emote
+    buildsystem: meson
+    sources:
+      - type: dir
+        path: "./"
```

Comment:
this will not work. There is no source.

Reply:
Indeed, I forgot to change the source to the git repository, I fixed it here: https://github.com/flathub/flathub/pull/4265/commits/38f1e8d2384e216a9ca2da1f78c29afd586cacd3